---
title: "Detection of protein-protein interactions by  bio-orthogonal fluorogenic proximity probes"
format:
  manuscript_template-pdf:
    keep-tex: true 
    link-citations: true
    cite-method: natbib
author:
  - name: Prabash Wijethunga
    affiliations:
      - name: Department of Immunology, Genetics & Pathology
        id: †
        department: Uppsala University
        city: Uppsala
        postal-code: SE-752 37
        country: SWEDEN
    attributes:
      equal-contributor: false
  - name: Alfred N. Larsson
    affiliations:
      - ref: †
    attributes:
      equal-contributor: false
  - name: Andreas Torell
    affiliations:
      - ref: †
    attributes:
      equal-contributor: false    
  - name: Luke Odell  
    affiliations:
      - name: Department of Medicinal Chemistry
        id: §
        department:  Uppsala University
        city: Uppsala
        postal-code: SE-752 37
        country: SWEDEN  
  - name: Daniel Fürth
    affiliations:
      - ref: †
      - name: SciLifeLab
        id: ‡
        department: Department of Immunology, Genetics & Pathology
        city: Uppsala
        postal-code: SE-752 37
        country: SWEDEN
    email: furth@scilifelab.uu.se 
    attributes:
      equal-contributor: false
      corresponding: true
abstract: |
  Detecting protein-protein interactions within cells is challenging. Transgenic approaches risk altering protein function via fluorescent tagging, while in situ methods lack in vivo compatibility. Here, we introduce fluorogenic probes with dual-tetrazine pegylated branched arms linked to xanthene dye. Activation requires both tetrazine arms to interact simultaneously with target proteins, enabling dual-substrate recognition. We applied our method to detect protein-protein interactions in both fixed and living cells, utilizing antibody conjugation for fixed cells and genetic code expansion for real-time detection in living cells. Our strategy ensures versatile applicability and seamless transition between fixed and living systems.
runningauthor: "One, Two, and Three"
significance: |
  Authors must submit a 120-word maximum statement about the significance of their research paper written at a level understandable to an undergraduate educated scientist outside their field of speciality. The primary goal of the significance statement is to explain the relevance of the work in broad context to a broad readership. The significance statement appears in the paper itself and is required for all research papers.
# author_contributions: "Please provide details of author contributions here."
conflict_of_interest: "Conflict of interest: D.F. serves on the scientific advisory board of Navinci Diagnostics AB."
#equal-contributor: "A.O.(Author One) contributed equally to this work with A.T. (Author Two) (remove if not applicable)."
#keywords: [protein-protein interaction, fluorogenic, microscopy]
#acknowledgements: | 
  #Please include your acknowledgments here, set in a single paragraph. Please do not include any acknowledgments in the Supporting Information, or anywhere else in the manuscript.
bibliography: bibliography.bib  
fontsize: "10pt"
classoption:
 - twocolumn
 - twoside
latex-clean: false
engine: knitr
---

Several techniques are available for detecting protein-protein interactions and pinpointing their cellular locations. These approaches can be broadly categorized into two main groups: transgenic fusion-protein techniques, including methods that involve fluorescent proteins [@pfleger_illuminating_2006; @hu_simultaneous_2003], and methods employing a combination of affinity reagents with fluorescence resonance energy transfer or readout by enzymatic detection and amplification techniques using oligonucleotides [@soderberg_direct_2006; @koos_proximity-dependent_2015; @yan_analysis_2003]. 

Transgenic techniques offer the distinct advantage of enabling live imaging, allowing for real-time monitoring of interactions within living cells [@luker_noninvasive_2002]. In contrast, the latter category of methods, utilizing affinity reagents and enzymatic amplification techniques, excels in detecting endogenous proteins where transgenic approaches are not feasible, such as in clinical samples [@soderberg_direct_2006].

What has been lacking is a fluorescent technique that operates independently of transgenic expression of fusion proteins, thus sidestepping potential disruptions in protein function due to tagging, while also addressing sensitivity and reliability challenges typically associated with multi-step enzymatic detection and amplification procedures. A unified detection method capable of functioning effectively in both fixed and living cells would bridge these two prominent methodological categories. 

In tandem with these challenges, proximity labeling techniques, despite their achievements [@Hung2014; @Roux2012; @Steensel2000], face a distinct issue. A persistent challenge lies in defining a precise spatial distance that signifies interaction between proteins. This limitation stems from the reliance on proximity-driven events without a well-defined threshold for spatial constraints. The ambiguity in interpreting proximity labeling data mirrors the limitations of traditional pulldown assays, posing an impediment to precisely delineating the spatial proximity necessary for accurate molecular interactions and contributing to the occurrence of false positives. This paper seeks to contribute to the field by addressing these dual challenges, providing a comprehensive solution that not only operates independently of transgenic expression but also offers precise control of spatial distances, akin to a molecular ruler. The integration of non-enzymatic and bio-orthogonal ligation chemistries with fluorogenic properties presents a promising solution to fulfill this critical need [@devaraj_bioorthogonal_2010; @Mboyi2021; @Kormos2018]. In this paper we developed and validated a fluorogenic dual-substrate recognition probe. We investigate its specificity in detecting protein-protein interactions, highlighting its capability to distinguish true interactions from mere proximity detection. Furthermore, we employ our probes for the detection of endogenous targets using conjugated antibodies and genetic code expansion for real-time monitoring of dynamic protein-protein interactions in live cells.

\begin{figure*}[bt!]
  \label{fig:01}
  \includegraphics[width=\textwidth]{figures/fig01.pdf}
  \caption{ \textbf{Fluorogenic Proximity (FluoroProx) Probes Enable Dual-Substrate Recognition. a)} Schematic of fluorogenic proximity probes. A branched PEG linker with three arms connects a xanthene-derivative dye to two tetrazines. The energy transfer and quenching of the dye are mitigated only upon reaction of the tetrazine with trans-cyclooctene conjugated antibodies, resulting in rapid fluorogenic activation. \textbf{b)} Quenching of AZDye488 and AZDye594 upon conjugation to PEG1-bis(PEG2-tetrazine), Tz, compared to unquenched, -. \textbf{c)} Emission spectra of AZdye488 and AZdye594 before and after bis-tetrazine quenching. \textbf{d)} Detection of the cMyc/Max transcription factor oncogene complex in PC3 cells. Primary cMyc and Max antibodies are detected by secondary antibodies conjugated to trans-cycloctene (TCO), which serves as the substrate for the branched tetrazine arm's inverse electron-demand reaction. Dual-substrate recognition results in complete unquenching of the fluorescent signal. \textbf{e)} Single primary antibody presence, either cMyc or \textbf{f)} Max, is insufficient to reverse the quenching effect caused by unreacted free tetrazine arms. \textbf{g)} Omission of secondary TCO-antibodies. Green: AZDye594 signal, magenta: Phalloidin-FITC, blue: DAPI. Scale bar: 20 μm. In the schematics of d-g, the indirect antibody staining is omitted for simplicity. \textbf{h)} Quantitative comparison of fluorescent signal with FluoroProx probes compared to traditional immunofluorescence for cMyc/Max in nucleus versus cytosol. Groups indicate presence of primary antibody. Neg., are negative controls, where all primary antibodies were ommitted, only secondary used (*** $P < 0.001$, n.s. $P > 0.05$). Error bars +/- 1 S.E.M. $n = 689$ cells}
\end{figure*}

# Fluorogenic Proximity Probes (FluoroProx)  {.unnumbered}

Here we present a method that involves the synthesis of fluorogenic probes, linking a xanthene-derivative dye to a branched dual-substrate recognition arm through poly(ethylene glycol) (PEG) linkers. The fluorescence of these probes only activates upon the simultaneous recognition of both arms by their respective targets. Branched PEG linker arms have long enabled researchers to synthesize compounds with multiple terminal functional groups generating superior properties in cellular uptake and biodistribution [@monfardini_branched_1995; @roberts_chemistry_2002]. These arms are structured to position two reactive groups at a precise distance from each other (**\hyperref[fig:S01]{Fig. S1a}**), facilitating catalysis of the second reactive group when brought into close proximity by the reaction of the first. We utilized branched PEGylation of a fluorophore to bring into close proximity two tetrazines (**\hyperref[fig:01]{Figure 1a}**). Tetrazines in these fluorogenic probes play a dual role. Firstly, they serve as diene groups, engaging in an inverse electron-demand Diels–Alder reaction with their target substrate containing a dienophile [@blackman_tetrazine_2008]. Secondly, tetrazines also function as fluorescence quenchers for the fluorophore (**\hyperref[fig:01]{Fig. 1b}**). The fluorophore can thus exist in three different states - a fully quenched state, a semi-quenched state and an unquenched state (**\hyperref[fig:01]{Fig. 1a}**).

Our system boasts a distinctive feature: its inherent bio-orthogonality, achieved through the seamless integration of xanthene-derivative dyes, PEG-linkers, and tetrazines. Notably, compatible with our system are rhodamine dyes with azetidine rings [@grimm_general_2015], which demonstrate outstanding photostability and cell permeability, rendering them highly suitable for use in living cells. Furthermore, PEG-linkers, in addition to augmenting cellular uptake, offer stability and \emph{in vivo} compatibility. Lastly, tetrazines play a pivotal role in bio-orthogonal inverse electron-demand Diels-Alder (iEDDA) “click reactions” [@blackman_tetrazine_2008], ensuring the overall compatibility of the system in living and fixed cells. The probes are easily synthesized with common molecular biology equipment through strain-promoted azide-alkyne cycloaddition conjugation of dibenzocyclooctyne-containing fluorophores, yielding a high product yield and eliminating the need for subsequent purification (**\hyperref[fig:S01]{Fig. S1a,c}**).

Prior studies have established that quenching efficiency of tetrazine dyes is significantly affected by the distance between the tetrazine and fluorophore chromophores [@wieczorek_rigid_2014]. 
Consequently, our initial investigation aimed to assess whether the PEG-linker of our probes introduced an excessive distance between these chromophores, potentially compromising 
fluorescent turn-on for practical applications. We evaluated quenching of both a green-fluorescent dye (AZDye488, $\textrm λ_{\textrm{abs}}$ 490, $\textrm λ_{\textrm{em}}$ 517 nm) as well as a red-fluorescent dye (AZDye594, $\textrm λ_{\textrm{abs}}$ 590 nm, $\textrm λ_{\textrm{em}}$ 613 nm). Despite the increased distance introduced by the PEG-arm, both dyes consistently demonstrate potent quenching capabilities (**\hyperref[fig:01]{Fig. 1b}**), yielding fluorescent quench emission ratios of `r 5.74` and `r 2.88`, respectively (**\hyperref[fig:01]{Fig. 1c}**). This highlights the system's capability to effectively quench different fluorophores with varying emission spectra, demonstrating the utility of multispectral detection with the system. Importantly, while the addition of the tetrazine-PEG-arm created a small bathochromic shift in the excitation peak by 6 and 3 nm in both dyes ($\textrm λ_{\textrm{abs}}$ 496, $\textrm λ_{\textrm{abs}}$ 593 nm) (**\hyperref[fig:S01]{Fig. S1b}**), the emission peak remained largely unaltered at $\textrm λ_{\textrm{em}}$ 515 nm and $\textrm λ_{\textrm{em}}$ 613 nm respectively (**\hyperref[fig:01]{Fig. 1c}**).

Next, we evaluated the system's performance by directly detecting protein-protein interactions in fixed cells. Our choice of target was the oncogene transcription factor complex cMyc/MAX (myc-associated factor X, MAX), selected due to its previous utility in evaluating and introducing the enzymatic proximity ligation assay to a wider audience [@soderberg_direct_2006]. Furthermore, the unambiguous subcellular localization of the cMyc/MAX complex, primarily within the nucleus, as opposed to each individual protein, which is localized in both the nucleus and cytosol (**\hyperref[fig:S02]{Fig. S2}**), makes it an ideal candidate for assessing the specificity of our method. 

```{r echo = FALSE}
#fluoroprox
data <- read.table('./data/panel1h.csv', header=TRUE, sep=',')
fp <- data[data$fluoroprox, ]

#primary = type of antibody primary, region = nucleus or cytosol
fp <- fp[fp$primary %in% c('Max', 'cMyc:Max'),]

mod1 <- aov(val ~ primary*region, data = fp)

mod1 <- anova( mod1 )

#if/icc, primary = type of antibody primary, region = nucleus or cytosol
ihc <- data[!data$fluoroprox, ]

#primary = type of antibody primary, region = nucleus or cytosol
ihc <- ihc[ihc$primary %in% c('Max', 'cMyc:Max'),]

mod2 <- aov(val ~ primary*region, data = ihc)

mod2 <- anova( mod2 )

#function to extract P-value
p.value <- function(mod, p_value_index) {
  mod_unlist <- unlist(mod)
  nfactors <- length(mod$Df) - 1
  p.values <- mod$`Pr(>F)`
  degree.of.freedoms <- c(mod$Df[p_value_index], mod$Df[nfactors+1])
  f.value <- mod$`F value`[p_value_index]
  f.value <- paste0('$F_{', degree.of.freedoms[1], ',', degree.of.freedoms[2], '} = ', round(f.value,2), '$, ')
  if (p_value_index <= length(p.values)) {
    p_value_numeric <- p.values[p_value_index]
    p_value_string <- ifelse(p_value_numeric < 0.0001, "_P_ < 0.0001", ifelse(p_value_numeric < 0.001, "_P_ < 0.001", ifelse(p_value_numeric < 0.01, "_P_ < 0.01", ifelse(p_value_numeric < 0.05, "_P_ < 0.05", "_P_ > 0.05"))))
    return(c(p_value_numeric, paste(f.value, p_value_string)))
  } else {
    stop("Invalid p-value index")
  }
}
``` 

Since rabbit-raised antibodies are easily accessible for diverse set of targets, we aimed to assess the feasibility of using two rabbit-raised antibodies in a single assay. Using antibodies from separate species could be limiting due to the comparatively smaller pool of available antibodies for non-rabbit species, potentially restricting the range of interactions that can be tested. Rabbit-raised primary antibodies for cMyc and MAX were targeted by secondary antibodies conjugated to the dienophile trans-cyclooctene (TCO, **\hyperref[fig:S01]{Fig. S1e-g}**). After washing, a quick incubation with the pegylated Y-shaped fluorogenic tetrazine probe generated a fluorescent signal restricted to the nucleus (**\hyperref[fig:01]{Fig. 1d}**). This strong fluorogenic turnon was only observed when both antibodies were present (**\hyperref[fig:01]{Fig. 1d-h}**, `r p.value(mod1, 3)[2]`). This is in stark contrast to ordinary immunofluorescence where combining antibodies did not generate an additative signal (`r p.value(mod2, 3)[2]`).
As expected the cMyc/MAX complex is localized in the nucleus because the heterodimer binds to DNA and acts as a transcription factor. In contrast, individual cMyc and MAX proteins, as visualized by standard immunofluorescence, can be found in both the nucleus and cytosol (**\hyperref[fig:01]{Fig. 1h}, \hyperref[fig:S02]{Fig. S2b-d}**). 

<!-- Importantly, Max is known to form homodimers that can act as transcription factors, but cMyc is unable to form homodimers and requires interaction with Max to form a hetereodimer that acts as a transcription factor. Our method specifically picked up on this distinction since a subset of cells with increased signal when Max primary antibodies was used alone while this was not seen when cMyc was used alone (**\hyperref[fig:01]{Fig. 1h}**). Hence, out method could be used to effectively screen for hetereo and homodimer pairs. -->

\begin{figure}[!t]
  \label{fig:02}
  \includegraphics[width=0.5\textwidth]{figures/fig02.pdf}
  \caption{ \textbf{FluoroProx Signal is Antigen-Specific. a)} Detection of cMyc:Max rabbit IgG primary antibodies. \textbf{b)} Negative control with same immunoglobulin isotype, rabbit IgG, as Max and cMyc. No significant  interaction with cMyc is detected. \textbf{c)} Isotype control, rabbit IgG, alone exhibits same pattern as cMyc:isotype control. Faint granulated signal in the nucleus. Scale bar: 20 μm. \textbf{d)} Quantification of average fluorescent signal intensity in cytosol, dark gray, and nucleus, light gray. }
\end{figure}


```{r echo = FALSE}
data <- read.table('./data/IgG.csv', header=TRUE, sep=',')
data$zscore <- scale(data$Mean)*10+50

cMycMax <- data$zscore[(!data$cytosol)&(data$groups == 'cMyc:Max')]

cMycIgG <- data$zscore[(!data$cytosol)&(data$groups == 'cMyc:isotype')]

mod1 <- t.test(cMycMax, cMycIgG)

isotype <- data$zscore[(!data$cytosol)&(data$groups == 'isotype')]

mod2 <- t.test(cMycIgG, isotype)

mod3 <- t.test(cMycMax, isotype)
``` 

\begin{figure*}[bt!]
  \label{fig:03}
  \includegraphics[width=\textwidth]{figures/fig03.pdf}
  \caption{ \textbf{FluoroProx Probes Detect Interactions rather than Proximity. a)} 
  Targeting of the E-cad/-cat protein interaction localized to the cell membrane by monoclonal antibodies and FluoroProx probes. \textbf{b)} Schematic showing possible false positive interaction displayed by tethering GFP to the plasma membrane with a palmitoylation signal and . Schematic of fluorogenic proximity probes. A branched PEG linker with three arms connects a xanthene-derivative dye to two tetrazines. The energy transfer and quenching of the dye are mitigated only upon reaction of the tetrazine with trans-cyclooctene conjugated antibodies, resulting in rapid fluorogenic activation. \textbf{b)} Quenching of AZDye488 and AZDye594 upon conjugation to PEG1-bis(PEG2-tetrazine), Tz, compared to unquenched, -. \textbf{c)} Emission spectra of AZdye488 and AZdye594 before and after bis-tetrazine quenching. \textbf{d)} Detection of the cMyc/Max transcription factor oncogene complex in PC3 cells. Primary cMyc and Max antibodies are detected by secondary antibodies conjugated to trans-cycloctene (TCO), which serves as the substrate for the branched tetrazine arm's inverse electron-demand reaction. Dual-substrate recognition results in complete unquenching of the fluorescent signal. \textbf{e)} Single primary antibody presence, either cMyc or \textbf{f)} Max, is insufficient to reverse the quenching effect caused by unreacted free tetrazine arms. \textbf{g)} Omission of secondary TCO-antibodies. Green: AZDye594 signal, magenta: Phalloidin-FITC, blue: DAPI. Scale bar: 20 μm. In the schematics of d-g, the indirect antibody staining is omitted for simplicity. \textbf{h)} Quantitative comparison of fluorescent signal with FluoroProx probes compared to traditional immunofluorescence for cMyc/Max in nucleus versus cytosol. Groups indicate presence of primary antibody. Neg., are negative controls, where all primary antibodies were ommitted, only secondary used (*** $P < 0.001$, n.s. $P > 0.05$). Error bars +/- 1 S.E.M. $n = 689$ cells}
\end{figure*}

# FluoroProx is Antigen- and Interaction-Specific.  {.unnumbered}

\begin{figure*}[bt!]
  \label{fig:04}
  \includegraphics[width=\textwidth]{figures/fig04.pdf}
  \caption{ \textbf{Live-cell Imaging of Protein-Protein Interactions. a)} Schematic of fluorogenic proximity probes. A branched PEG linker with three arms connects a xanthene-derivative dye to two tetrazines. The energy transfer and quenching of the dye are mitigated only upon reaction of the tetrazine with trans-cyclooctene conjugated antibodies, resulting in rapid fluorogenic activation. \textbf{b)} Quenching of AZDye488 and AZDye594 upon conjugation to PEG1-bis(PEG2-tetrazine), Tz, compared to unquenched, -. \textbf{c)} Emission spectra of AZdye488 and AZdye594 before and after bis-tetrazine quenching. \textbf{d)} Detection of the cMyc/Max transcription factor oncogene complex in PC3 cells. Primary cMyc and Max antibodies are detected by secondary antibodies conjugated to trans-cycloctene (TCO), which serves as the substrate for the branched tetrazine arm's inverse electron-demand reaction. Dual-substrate recognition results in complete unquenching of the fluorescent signal. \textbf{e)} Single primary antibody presence, either cMyc or \textbf{f)} Max, is insufficient to reverse the quenching effect caused by unreacted free tetrazine arms. \textbf{g)} Omission of secondary TCO-antibodies. Green: AZDye594 signal, magenta: Phalloidin-FITC, blue: DAPI. Scale bar: 20 μm. In the schematics of d-g, the indirect antibody staining is omitted for simplicity. \textbf{h)} Quantitative comparison of fluorescent signal with FluoroProx probes compared to traditional immunofluorescence for cMyc/Max in nucleus versus cytosol. Groups indicate presence of primary antibody. Neg., are negative controls, where all primary antibodies were ommitted, only secondary used (*** $P < 0.001$, n.s. $P > 0.05$). Error bars +/- 1 S.E.M. $n = 689$ cells}
\end{figure*}

While it is promising that primary antibodies from the same species can be used, this approach presents the potential for non-specific interactions and cross-reactivity. There is a possibility of these antibodies binding to each other due to their shared, similar species-specific epitopes, instead of binding to their intended targets. Such interactions can lead to false-positive results and inaccuracies in data interpretation. To enhance the evaluation of method specificity, we replaced the Max antibody with a rabbit polyclonal IgG isotype control (**\hyperref[fig:02]{Fig. 2}**). We  quantified the fluorescent intensity in the nucleus as well as in the cytosol and samples where both cMyc and Max primary antibodies were used conjoint in staining (_M_ = `r round(mean(cMycMax),2)`, _SD_ = `r round(sd(cMycMax),2) `) had twice the fluorescent intensity (_t_ = `r round(mod1$statistic, 2)`, _P_ < `r round(mod1$p.value, 6)`) compared to samples that were stained by cMyc and isotype control (**\hyperref[fig:02]{Fig. 2b}**, _M_ = `r round(mean(cMycIgG),2)`, _SD_ =  `r round(sd(cMycIgG),2)`) or isotype control alone (**\hyperref[fig:02]{Fig. 2c}**, _t_ = `r round(mod3$statistic, 2)`, _P_ < `r round(mod3$p.value, 6)`). Importantly, isotype control alone (_M_ = `r round(mean(isotype),2)`, _SD_ = `r round(sd(isotype),2)`) did not differ significantly from cMyc together with isotype control (_t_ = `r round(mod2$statistic, 2)`, _P_ = `r round(mod2$p.value, 2)`), indicating that cross-reactivity and unspecific binding is less of a concern.

<!--

\begin{figure}[!t]
  \label{fig:03}
  \includegraphics[width=0.5\textwidth]{figures/fig03.pdf}
  \caption{ \textbf{Detection of E-cadherin and $\beta$-catenin protein-protein interaction. a)} Detection of E-cad:$\beta$-cat, rabbit IgG and mouse IgG1 primary antibodies. \textbf{b)} Negative control without E-cadherin rabbit IgG primary antibody. \textbf{c)} Negative control without $\beta$-catenin mouse IgG1 primary antibody. Scale bar: 10 μm. Green: FluoroProx AZdye594. Magenta: Phalloidin-Alexa488, Blue: DAPI. }
\end{figure}

-->

Having observed that our method specifically detects protein-protein interactions subcellularly localized to the cell nucleus we decided to test the method on detection a protein-protein interaction pair localized to a different subcellular compartment. The membraine domain pair E-cadherin and β-catenin is a highly studied interactions due to its role in epithelial-mesenchymal transition. We targeted each protein using monoclonal primary antibodies with epitopes close to the site of interaction (**\hyperref[fig:03]{Fig. 3a}**). A membrane localized fluorescent turn-on signal was only observed when both antibodies were used (**\hyperref[fig:S03]{Fig. S3}**, **\hyperref[fig:03]{Fig. 3e}**). Since we now had a pair of differentially localized protein-protein interactions in cMyc/Max (nucleus) and E-cad/β-cat (membrane) we decided to use these two settings to evaluate our methods robustness towards spurious proximity-based interactions rather than true protein-protein interaction. By introducing a green fluorescent protein (GFP) tagged with N-terminal localization signals (palmitoylation signal from GAP43 or NLS from SV40) we could either localize GFP to the vicinity of E-cad/β-cat at the lipidmembrane (**\hyperref[fig:03]{Fig. 3b}**) or near cMyc/Max in the nucleus (**\hyperref[fig:03]{Fig. 3d}**).

<!-- In addition, since commercially available unconjugated secondary antibodies are difficult to reliably conjugate with a specific degree of labeling of one molecule per antibody we used secondary single domain antibodies (sdAb), or nanobodies, with a N-terminal cysteine tag that we conjugated using cysteine/maleimide conjugation to TCO-PEG3-maleimide. Using these nanobodies we observed a reliable interaction signal localized to the cell membrane only when both antibodies were present (**\hyperref[fig:03]{Fig. 3}**). -->


# Live Imaging using Genetic Code Expansion {.unnumbered}

For live imaging, we aimed to site-specifically label a protein capable of forming homodimers. Omomyc is a synthetic protein developed to target and inhibit the activity of cMyc and its interaction with Max [@Soucek1998]. Omomyc works by binding to cMyc and preventing its regulation of transcription and cell proliferation, which are involved in promoting cancerous growth. This inhibition of cMyc activity has shown promise in studies as a potential therapeutic approach for cancer treatment. When Omomyc is present, it forms heterodimers with cMyc, thereby sequestering cMyc away from DNA. Additionally, it creates transcriptionally inactive homodimers and heterodimers with Max, which occupy E-boxes, leading to the inhibition of transcription of cMyc targets. Therefore, we decided to tag the Omomyc/Omomyc dimers by site-directed labeling through genetic code expansion (**[Fig. 4a](#fig:04)**). A tamoxifen-inducible Omomyc plasmid [@Savino2011] was labeled for genetic code expansion by site-directed mutagenesis of a glycine codon at position 91 into an amber codon (OmomycTAG~86~-ER, **[Fig. 4b](#fig:04)**). We verified that the Omomyc synthetic protein indeed inhibited cMyc/Max interaction as measured by FluoroProx antibody staining with cMyc/Max as the target (**[Fig. 4c,d](#fig:04)**). Co-transfection of the OmomycTAG~91~-ER plasmid with a plasmid expressing a tRNA and its cognate aminoacyl-tRNA-synthetase pair [@Seidel2017] together with the non-canonical amino acid (ncAA) azido-phenylalanine (azido-Phe or AzF) enabled site-directed labeling and strong nuclear localization when counter-stained with both a DNA and lipid membrane stain (**[Fig. 4e-g](#fig:04)**). In conclusion, our study serves as a proof-of-concept for directly visualizing protein-protein interactions in living cells through the site-specific labeling of Omomyc/Omomyc dimers using genetic code expansion.



# Discussion  {.unnumbered}

The challenge of detecting protein-protein interactions within cells has long been a bottleneck in understanding cellular processes. Here we introduces a novel approach to tackle this challenge by employing fluorogenic probes with dual-tetrazine pegylated branched arms linked to xanthene dye. The design of these probes addresses the limitations associated with existing methods, providing a promising avenue for in vivo compatibility without altering protein function via fluorescent tagging, while also being compatible with detection of endogenous proteins.

In addition to its current applications, the technology introduced in this study holds great promise for future advancements in the direct tracking of protein-protein interactions as well as protein-ligand interactions within living organisms. The dual-tetrazine pegylated branched arms linked to xanthene dye offer a unique opportunity to study protein-ligand interactions in real-time, providing a potential breakthrough in understanding drug kinetics and interactions directly \emph{in vivo}. The non-invasive nature of the fluorescent probes mitigates concerns associated with altering protein function or cellular processes, making it an ideal candidate for investigating the dynamic interplay between drugs and their target proteins within the complex biological environment. This capability to observe and quantify protein-ligand interactions in living systems has profound implications for drug development, allowing researchers to gain insights into the temporal aspects of drug binding, dissociation, and overall pharmacokinetics. The presented technology, with its dual-substrate recognition mechanism, not only expands our toolbox for studying interactions but also opens up new avenues for advancing our understanding of drug behavior in the intricate milieu of living organisms, ultimately contributing to more informed and targeted drug design strategies.



<!--  indicating that the short PEGylated spacer arm provides a necessary maximum distance to be able to distinguish true protein interactions from mere proximity. -->

# Material & Methods {.unnumbered}

## Image analysis {.unnumbered}

Raw images were acquired as 16-bit z-stacks but analysis was done on maximum intensity projections in 16-bit raw tiff format. For cell segmentation 

## Statistical analysis {.unnumbered}

All statistical computations were done with R the statistical programming language. Fluorescent intensity was normalized between replicates to a mean of 50 and standard deviation of 10 by t-score transformation:

\begin{equation}
t = 10\left(\frac{x - \bar{x}}{s}\right) + 50
\end{equation}

\begin{equation}
\begin{aligned}
\text{Where:} \\
& t \quad \text{: t-score} \\
& x \quad \text{: average pixel intensity of object} \\
& \bar{x} \quad \text{: sample mean} \\
& s \quad \text{: sample standard deviation}
\end{aligned}
\end{equation}

Statistical tests performed were either two-sided t-tests or one-way ANOVA. 

## End-to-end distance polymer linker simulation {.unnumbered}

Chemical structure was drawn using ChemDraw and saved as `.cdxml` files. To convert the ChemDraw file into GROMACS-compatible `.pdb` file we used open-babel with the following settings:

```
obabel input.cdxml -O output.pdb --gen3d -p 7.4
```

Distances were then calculated through:

```
gmx distance -f trajectory.trr -s topology.tpr 
-n index.ndx -select 'group1 "end1" group2 "end2"' 
-oav distance.xvg
```

## Cell culture  {.unnumbered}
Prostate cancer cells (PC3) were cultivated in Advanced DMEM (12491015, Thermofischer Scientific) supplemented with 10% fetal bovine serum (10106-147, Thermofischer Scientific), 5% Pen-Strep (15140-122, Thermofischer Scientific) and 5% L-glutamine (25030-024, Thermofischer Scientific). The cells were incubated at 37°C under a 5% CO2 atmosphere.

## Fluorogenic probe synthesis  {.unnumbered}

N-(Azido-PEG1)-N-bis(PEG2-Methyltetrazine-propylamine)
(BP-28748, Broad Pharma Inc.) was dissolved in anhydrous DMSO (276855, Sigma-Aldrich Gmbh) to a stock concentration of 10 mM. Likewise, fluorophore of choice (AZdye488-DBCO or AZdye594-DBCO, 1278-1 or 1298-1 Click Chemistry Tools Inc.) was dissolved in DMSO to a a stock concentration of 10 mM. Initially the pegylated bis-tetrazine arms were conjugated to fluorophore of choice through azide/DBCO Strain-promoted Azide-Alkyne Click (SPAAC) reaction in DMSO at 1:1 molar equivalents for 24h in room temperature. Conjugation was verified by fluorescent quenching of the fluorophore by the tetrazines as well as liquid chromatography–mass spectrometry (LC–MS).

## Antibody conjugation {.unnumbered}

Antibody conjugation was performed using TFP-esters according to the following protocol. First, antibody purification was carried out to remove interfering compounds if present such as BSA, azide, or glycine often added by the manufacturer. This involved washing the antibody in 1X PBS using Amicon ultra 10K 0.5mL columns, followed by collection of the purified antibody and determination of its concentration using a NanoDrop spectrophotometer at 280 nm.

For ester preparation,  TRP-esters (TCO-PEG4-TFP, BP-40298, Broad Pharma Inc.) were dissolved in anhydrous DMSO to a concentration of 10 mg/mL. The antibody was then labeled at 2 mg/mL in 0.5 M carbonate/bicarbonate buffer (pH 8.75) at various molar ratios of ester to antibody ($3\times$, $9\times$, $15\times$). After incubation at room temperature for 1h in the dark, remaining esters were quenched by the addition of 1M Tris buffer (pH 8.0) to a final concentration of 100 mM and incubated on ice for 15 min.

The labeled antibody conjugates were purified and concentrated using Zeba Spin Desalting Columns 40K MWCO (11796436, ThermoFisher Inc.) and Amicon ultra 30K 0.5mL centrifugation column (UFC5030, Merck GmbH). Excess esters were removed through multiple wash steps with 1X PBS, and the final conjugates were collected and their concentrations and degree of labeling was determined by NanoDrop and SDS-PAGE analysis. The resulting antibody conjugates were diluted to a final concentration of 2 mg/mL with 1X PBS and stored at 4 degrees Celsius for subsequent experiments.
Full protocol and all reagents is accessible here: \url{https://www.furthlab.xyz/antibody_conjugation}.

## Fluorescent spectrophotometry {.unnumbered}

FluoroProx probes at a stock concentration of 5 mM were diluted in 1xPBS pH 7.5 to a concentration of 2 uM for a total volume of 2.5 mL in polystyrene disposable cuvettes with four clear faces (634-8530, VWR) and emission was measured on a Fluorolog SPEX TCSPC Horiba fluorescence spectrophotometer (ex. wavelength 488 and 594 nm, 0.1 sec. integration time, 1 nm slit). Absorbance was measured by UV-Vis spectrophotometry on a Denovix DS-11 spectrophotometer.

## Liquid chromatography–mass spectrometry (LC–MS) {.unnumbered}

LC-MS analysis was carried out using electrospray ionization (ESI) and a C18 column (50×3.0 mm, 2.6 μm particle size, 100 Å pore size) with acetonitrile/water in 0.05% aqueous formic acid as mobile phase.

_**MS(ESI)**_, m/z calc’d for C~80~H~86~N~18~O~20~S~2~: 1683.8; found 1697.7 [M+H]^+^, 849.0 [M+2H]^2+^.

_**MS(ESI)**_, m/z calc’d for C~94~H~105~N~18~O~20~S~2~: 1871.1; found 1885.7 [M+H]^+^, 943.1 [M+2H]^2+^.

FluoroProx488 _"AZDye488-DIBOT-PEG1-N-bis(PEG2-Tz)"_ <br>

**6-amino-9-\{2-carboxy-4-[(3-\{3-[13-\{3-[4-(6-methyl-1,2,4,5-tetrazin-3-yl)phenoxy\]propyl\}carbamoyl\)-5-(2-\{2-\{2-({3-\{3-[4-(6-methyl-1,2,4,5-tetrazin-3-yl)phenoxy\]propyl\}carbamoyl\)ethoxy\}ethoxy\}ethyl\)-3,8,11-trioxa-5-azatridecan-1-yl\]-3,4,5,13-tetraazatetracyclo[13.4.0.0^2,6^.0^7,12^]nonadeca-1(19),2(6),4,7,9,11,15,17-octaen-13-yl\}-3-oxopropyl\}carbamoyl\]phenyl\}-3-iminiumyl-5-sulfo-3H-xanthene-4-sulfonate**

![](./figures/fig_S_structureAZdye488.pdf)

FluoroProx594 _"AZDye594-DIBOT-PEG1-N-bis(PEG2-Tz)"_ <br>

**6-\{2-carboxy-4-\[3-\{3-\[13-\{3-\[4-(6-methyl-1,2,4,5-tetrazin-3-yl)phenoxy\]propyl\}carbamoyl\)-5-\{2-\{2-\{3-\[4-(6-methyl-1,2,4,5-tetrazin-3-yl)phenoxy\]propyl\}carbamoyl\}ethoxy\}ethoxy\}ethyl\)-3,8,11-trioxa-5-azatridecan-1-yl\]-3,4,5,13-tetraazatetracyclo[13.4.0.0^2,6^.0^7,12^]nonadeca-1(19),2(6),4,7,9,11,15,17-octaen-13-yl\}-3-oxopropyl\}carbamoyl\]phenyl\}-1,2,2,10,10,11-hexamethyl-4,8-bis(sulfonatomethyl)-2,10-dihydro-1H-13-oxa-1,11-diazapentacen-11-ium**

![](./figures/fig_S_structureAZdye594.pdf)

## Immunofluorescence {.unnumbered}
$7.5 \times 10^3$ PC3 cells were seeded on 35 mm glass bottom dishes (81218, Ibidi). Once the cells reached a confluence of 70%, the growth medium was replaced with 1X PBS (AM9625, Thermofischer scientific) and washed three times in ice cold PBS. The cells were then fixed with PFA (4% v/v Formaldehyde; 28908, Thermofischer Scientific) in 1X PBS for 15 minutes. After fixation cells were washed three times in PBST Triton-X100 ( 0.25% v/v Triton-X100; T8787, Sigma-Aldrich GmBh in 1X PBS) and permeabilized at room temperature for 10 minutes. To remove all traces of  detergent cells were washed three times in 1X PBS. 

Blocking was performed with a goat serum blocking medium (1 % v/v Goat serum; GOA-1A, Capricorn Scientific GmbH, diluted in 1X PBS with 0.1% v/v Tween-20; P9416, Sigma-Aldrich GmBh) for an hour at room temperature. Following this, the blocking medium was washed three times with PBST Tween-20 (0.1% v/v, P9416, Sigma-Aldrich GmBh , in 1X PBS; AM9625, Thermofischer Scientific ) with five minutes of incubation per wash step. The primary antibodies (C-myc Recombinant Rabbit Monoclonal antibody; lot: 2561984, 27H46L35, Thermo fischer Scientific,  Rabbit polyclonal Anti-Max antibody; log: A115844, HPA003474, Atlas Antibodies AB) were then diluted 1:500 in 0.1% (w/v) BSA (Bovine Serum Serum Albumin; A9647, Sigma-Aldrich Co in 1X PBS; AM9625, Thermofischer Scientific and 0.1% v/v Tween-20;P9416, Sigma-Aldrich GmBh) and incubated overnight at 4°C on a plate shaker with 300 RPM.

The following day, the cells were washed three times with PBST Tween-20 with five minutes of incubation at room temperature each. Secondary antibodies (Alexa Fluor 488 Chicken Anti-Rabbit IgG (H+L); Lot: 2549869, A21441, Thermo Fischer Scientific) at 1:1000 dilution were incubated in PBST Tween with 0.1% (w/v) BSA. Secondary antibodies were incubated in room temperature for 1 hour. The cells were once again washed in PBST Tween with the same protocol as above, then thoroughly washed in 1X PBS.

\begin{table*}[htbp]
    \centering
    \caption{Antibody Information}
    \begin{tabular}{@{}lllllll@{}}
    \toprule
    Antibody & Vendor & Catalog Number & LOT Number & Dilution & Host & Isotype & Clonality \\
    \midrule
    Alexa Fluor 488 Chicken Anti-Rabbit IgG (H+L) & Thermo Fischer Scientific & A21441 & 2549869 & 1:1000 & Chicken & IgY & Polyclonal \\
    Anti-Target2 & XYZ Inc. & cd67890 & LOT456 & 1:500 & Rabbit & IgG & Polyclonal \\
    \bottomrule
    \end{tabular}
    \label{tab:antibody}
\end{table*}

## Fluorogenic Dual-Substrate Recognition Assay   {.unnumbered}

The experimental procedure closely paralleled the methods employed for immunohistochemistry, with a key distinction being the utilization of a goat anti-rabbit secondary antibody (31210, ThermoFisher) conjugated to TCO-PEG4 (1398-2, Click Chemistry Tools Inc.) during the secondary incubation step. After washing, cells were exposed to FluoroProx probes at a concentration of 200 nM in PBS with XnM of phallodin-FITC and XnM DAPI for 20 minutes at room temperature, covered in foil. Cells were washed for three quick rounds of in PBST Tween-20 followed by three 5 minute PBST washes at 37°C and subsequently stored overnight in PBST at 4°C awaiting imaging. 

## Image acquisition and processing.  {.unnumbered}

Cells were imaged on a Nikon TE2 equipped with a CrestOptics XLight V3 spinning disc confocal. A fluorescent light source was provided by Lumencor’s CELESTA Quattro Light Engines with an arrays of 5 individually addressable solid-state lasers (Lumencor CELESTA Quattro nIR 5ch with Despeckler, 405/12, 476/12, 545/12, 637/12 and 748/12) and images were acquired with a Kinetix sCMOS Camera (Photometrics) through a oil immersion 100x objective. The spinning disc was equipped with a dichroic mirror 

## Plasmids  {.unnumbered}

Translocation of fluorophores to the nucleus was achieved by tagging the N-terminal of GFP with Simian Vacuolating Virus 40 (SV40) Nuclear Localisation Signal, CAG-NLS-GFP1 (Addgene plasmid #104061) was a gift from Viviana Gradinaru (California Institute of Technology). The cell membrane reporter used was N-terminal tagged GFP with Palmitoylated Growth Associated protein 43 (GAP-43), pCAG-mGFP2 (Addgene plasmid #14757) was a gift from Connie Cepko (Harvard Medical School). Interactions between β-Catenin and E-cadherin was studied through E-cadherin-GFP3 (Addgene plasmid # 28009) was a gift from Jennifer Stow (University of Queensland). Aminoacyl-tRNA synthetase and non-sense tRNA for incorporation of p-azido-phenylalanine was translated from pIRE4-Azi4 (Addgene plasmid # 105829), which was a gift from Irene Coin (Leipzig University) and the Tamoxifen-inducible fusion-protein of Omomyc, pCS Omomyc-Mer (Unpublished) (Addgene plasmid # 113170) was a gift from Sergio Nasi (CNR-IBPM).

## Plasmid Purification  {.unnumbered}

Plasmid-carrying bacterial cells were transferred from their respective agar stabs onto agar plates containing 100µg/mL of Ampicillin or 50µg/mL of Kanamycin and incubated overnight at 37℃ (DH5alpha) and 27℃ (NEB stable). Colonies of bacteria were subsequently harvested and cultured in 5ml of liquid LB medium overnight. The tubes containing the cultured cells were centrifuged at max RPM for 1 min after which, the supernatant was removed and the pellet resuspended in in an aqueous buffer of 50mM Tris HCl (ph 8), 10mM EDTA and 100µg/ml RNAse A. Lysis of bacterial cells was achieved using an alkaline solution containing 200mM NaOH with 1%SDS (v/w) diluted in nuclease-free water. After incubation at RT for 5 min, 3M Sodium Acetate was added prior to centrifugation at max RPM for 10 min. The supernatant was transferred to new tubes and the plasmid DNA was precipitated using cold 2-propanol on ice for 5 min before being centrifuged at max RPM for 5 min. Having discarded the supernatant, the pellet was washed with 70% ethanol before undergoing another centrifugation at max RPM for 5 min. After the supernatant was discarded and the pellet dried from any residual ethanol, the plasmid DNA (pDNA) was resuspended in nuclease-free water NF-H2O. The results of the pDNA-extraction was subsequently confirmed by Gel-Electrophoresis on 1% Agarose gel, stained with 1µl/mL SYBR™ Safe DNA Gel Stain (10,000X).

## Transfection  {.unnumbered}

PC3 cells were seeded and incubated overnight in Advanced DMEM containing 10% FBS, 100 IU/mL Penicillin-Streptomycin () and X L-Glutamate () at 37℃ in an atmosphere of 5% CO2. Upon reaching 70% confluency, cells were washed with 1xPBS before being detached by treatment with 5% Trypsin. After harvesting, the cells were concentrated using centrifugation and the pellet was subsequently resuspended in 1ml of culture medium. The transfection was carried out using Amaxa® Cell Line Nucleofector® Kit V. Approximately 106 cells (per reaction) were transferred to new 1.5ml tubes and centrifuged at max RPM for 10 min after which, the supernatant was removed completely. The cell-pellet was suspended in 100 µl Cell Line Nucleofector® Solution V and 2 µg of pDNA was added to the mixture. The entire sample was then transferred over to transfection cuvettes and placed in the cuvette holder of the machine (Amaxa® Nucleofector II). Electroporation was carried out using the cell appropriate program (T-013) after which, the solution was gently transferred from the cuvettes and plated onto 8-well chambers (CellVis, Mountain View, CA, Cat. #C8-1.5H-N).

## Genetic Code Expansion  {.unnumbered}

Site-Directed Mutagenesis (SDM) was performed on the pCS Omomyc-Mer plasmid (a gift from Sergio Nasi, Addgene plasmid #113170; http://n2t.net/addgene:113170; RRID:Addgene_113170)) to introduce an amber codon on position 86 of the OmoMYC insert. The primers used to modify the plasmid (Fwd: `CTACGGAACTCTTGTGCGTAA`; Ref: `CTATTCAAGTTTGTGTTTCAACTG`) were ordered from Integrated DNA Technologies (Leuven) at 25 nmole scale and reconstituted into nuclease-free water to a 100 µM stock concentration. The PCR was carried out using the Q5 Site-Directed Mutagenesis kit (New England Biosciences) for 25 µl reaction containing 25 ng of template plasmid, 12.5 µl Q5 Hot Start High-Fidelity Master Mix (2x), 9µl nuclease-free water and 1.25 µl of 10 µM forward and reverse primers. The thermal cycle program included 25 cycles consisting of 98℃ for 10 sec., 61℃ 30 sec. and elongation at 72℃ for 1 min 45 sec. Products were verified by gel electrophoresis on 1% agarose gel before re-circularizing the plasmid using KLD enzyme mix (New England Biosciences). The modified plasmid was subsequently transformed into competent bacterial cells (E.coli) through electroporation and underwent selective cultivation on LB-plates containing 100 µg/mL Ampicilin. Single colonies were transferred to liquid LB for enrichment following pDNA isolation as described earlier. Samples of plasmid from monoclonal origin were sent for Sanger sequencing (Eurofins Scientific) to validate the nucleotide substitution.

## Live Imaging of GCE-modified PC3 cells {.unnumbered}

The medium containing ncAA was removed from the plates and the cells were washed with DMEM to remove any excess amino acids. TCO-PEG4-DBCO was diluted in DMEM to a final concentration of 200 nM before being added to each sample and incubated for 45 min. The cells were then washed in DMEM three times over 45 min. FluoroProx probe (Tz-BiS-PEG3-AzDye594) 200 nM in DMEM with 1X Probenecid (Invitrogen, Cat#P36400) was subsequently added to each well and incubated for 1h followed by another wash-step as described above. Counterstaining of the nucleus and membrane was performed 30 min before imaging the cells using 2 µM SYTO™ Deep Red Nucleic Acid Stain (Invitrogen, Cat#S34900) and 0.1µM MemBright 640 (?) in DMEM with 1X Probenecid respectively for 30 min. The samples were rinsed once before replacing the DMEM with Leibowitz’s L-15 Medium (Gibco, Cat#11415064) containing 10% FBS, 100 IU/mL Penicillin-Streptomycin to allow for long term microscopy without CO2 equilibration.



\newpage
\renewcommand\thefigure{S\arabic{figure}} 
\setcounter{figure}{0} 

\begin{figure*}
\label{fig:S01}
\centering
  \includegraphics[width=0.9\textwidth]{figures/figS01.pdf}
  \caption{ \textbf{Synthesis of FluoroProx Probes. a)} Strain-promoted azide-alkyne cycloaddition (SPAAC) between N-(Azido-PEG1)-N-bis(PEG2-Methyltetrazine-propylamine) (BP-28748, Broad Pharma Inc.) 
and DBCO (dibenzocyclooctyne) labeled fluorophore (1278, Click Chemistry Tools Inc.) performed in equimolar concentrations in anhydrous DMSO at room temperature overnight.
\textbf{b)} Absorbance spectrum (excitation) of AZDye488-DBCO and AZDye594-DBCO before and after conjugation to N-(Azido-PEG1)-N-bis(PEG2-Methyltetrazine-propylamine).
\textbf{c)} Liquid chromatography–mass spectrometry (LC–MS) chromatogram of reactants (AZDye488-DBCO, AZDye594, and N-(Azido-PEG1)-N-bis(PEG2-Methyltetrazine-propylamine)) and products (AZDye488-DIBOT-PEG1-N-bis(PEG2-Tz) and AZDye594-DIBOT-PEG1-N-bis(PEG2-Tz)). Structure of individual molecules shown to the right for clarity.
\textbf{d)} Mass spectrogram of AZDye488-DIBOT-PEG1-N-bis(PEG2-Tz) and AZDye594-DIBOT-PEG1-N-bis(PEG2-Tz) from LC-MS in c).
\textbf{e)} TCO-PEG4-TFP Ester used to label primary aliphatic amines on secondary antibodies.
\textbf{f)} SDS-PAGE of TCO-conjugated antibodies and tetrazine-AZdye488 labeling. Lane 1 commercially conjugated goat secondary antibody. Lane 2: antibody to TCO-PEG4-TFP molar ratio of 1:3. Lane 3: molar ratio of 1:9. Lane 4: molar ratio of 1:15. Lane 5: tetrazine-AZdye488 alone. 
\textbf{g)} Degree of Labeling (DOL) of TCO-PEG4 conjugated antibodies.
}
\end{figure*}

\newpage

\begin{figure*}
\label{fig:S02}
\centering
  \includegraphics[width=\textwidth]{figures/figS02.pdf}
  \caption{ \textbf{Comparison of IF/ICC to FluoroProx. a)} Immunofluorescence of cMyc using monoclonal rabbit primary antibody visualized by polyclonal goat Alexa488-conjugated secondary antibody (green). Counter stained with DAPI (blue) and Phalloidin-iFluor594 (magenta). 
\textbf{b)} Absorbance spectrum (excitation) of AZDye488-DBCO and AZDye594-DBCO before and after conjugation to N-(Azido-PEG1)-N-bis(PEG2-Methyltetrazine-propylamine).
\textbf{c)} Liquid chromatography–mass spectrometry (LC–MS) chromatogram of reactants (AZDye488-DBCO, AZDye594, and N-(Azido-PEG1)-N-bis(PEG2-Methyltetrazine-propylamine)) and products (AZDye488-DIBOT-PEG1-N-bis(PEG2-Tz) and AZDye594-DIBOT-PEG1-N-bis(PEG2-Tz)). Structure of individual molecules shown to the right for clarity.
\textbf{d)} Mass spectrogram of AZDye488-DIBOT-PEG1-N-bis(PEG2-Tz) and AZDye594-DIBOT-PEG1-N-bis(PEG2-Tz) from LC-MS in c).
\textbf{e)} TCO-PEG4-TFP Ester used to label primary aliphatic amines on secondary antibodies.
\textbf{f)} SDS-PAGE of TCO-conjugated antibodies and tetrazine-AZdye488 labeling. Lane 1 commercially conjugated goat secondary antibody. Lane 2: antibody to TCO-PEG4-TFP molar ratio of 1:3. Lane 3: molar ratio of 1:9. Lane 4: molar ratio of 1:15. Lane 5: tetrazine-AZdye488 alone. 
\textbf{g)} Degree of Labeling (DOL) of TCO-PEG4 conjugated antibodies.
}
\end{figure*}

\newpage
